include/eve/detail/kumi.hpp

```diff
@@ -485,6 +486,47 @@ namespace kumi
       (std::make_index_sequence<sizeof...(Ts)>());
     }
 
+    template<product_type Other>
+    friend constexpr auto operator<(tuple const &lhs, Other const &rhs) noexcept
```

Comment:
I thought you were dead set against this?
If you are on board with it here, I'd suggest to just generate less.
I don't think it's a bad idea to have a default less.

I know you feel different and we can do that too, but maybe be consistent?

Reply:
Beware, this is inside KUMI, not inside EVE.
KUMI actually has an issue about this (https://github.com/jfalcou/ofw/issues/20).

What I am is being deadset on having `operator<` on user-defined type defined by default because the user knows if their own type is OK to be ordered.

On tuple with no semantic, it is ok to have it.